src/test/java/teammates/test/TestProperties.java

```diff
@@ -39,7 +39,7 @@ private TestProperties() {
                 prop.load(testPropStream);
             }
 
-            IS_SNAPSHOT_UPDATE = Boolean.parseBoolean(prop.getProperty("test.snapshot.update", "false"));
+            IS_SNAPSHOT_UPDATE = Boolean.parseBoolean(prop.getProperty("test.snapshot.update", "true"));
```

Comment:
Let's not change the default value for this! Snapshots should be checked first to ensure changes are expected before running the update. Do revert the changes